Description: The test for file rotation may take too long on Docker and fails
--- a/internal/rotate/file_writer_test.go	2021-05-21 09:11:21.000000000 -0700
+++ b/internal/rotate/file_writer_test.go	2021-08-19 10:30:19.000000000 -0700
@@ -29,14 +29,16 @@
 func TestFileWriter_TimeRotation(t *testing.T) {
 	tempDir, err := ioutil.TempDir("", "RotationTime")
 	require.NoError(t, err)
-	interval, _ := time.ParseDuration("1s")
+	// Rotate at 10s intervals since Docker may take longer than 1s to
+	// run, leading to incorrect test results
+	interval, _ := time.ParseDuration("10s")
 	writer, err := NewFileWriter(filepath.Join(tempDir, "test"), interval, 0, -1)
 	require.NoError(t, err)
 	defer func() { writer.Close(); os.RemoveAll(tempDir) }()
 
 	_, err = writer.Write([]byte("Hello World"))
 	require.NoError(t, err)
-	time.Sleep(1 * time.Second)
+	time.Sleep(10 * time.Second)
 	_, err = writer.Write([]byte("Hello World 2"))
 	require.NoError(t, err)
 	files, _ := ioutil.ReadDir(tempDir)
